source/about-compatibility.txt

```diff
@@ -0,0 +1,57 @@
+.. _about-driver-compatibility:
+
+============================
+MongoDB Compatibility Tables
+============================
+
+We include compatibility tables for each version of the driver to guide
+your decisions on what versions you need to ensure your environment
+remains fully operational.
+
+We maintain the following tables for each driver:
+
+- :ref:`MongoDB compatibility <mongodb-compatibility-tables>`
+- :ref:`Language compatibility <language-compatibility-tables>`
+
+Read the sections below for detailed explanations of each table.
+
+.. _mongodb-compatibility-tables
+
+MongoDB Compatibility Tables
+----------------------------
+
+In the **MongoDB Compatibility** tables, the columns are labeled with
+versions of MongoDB server and the rows are labeled with major release
+versions of the driver.
+
+The check marks (✓) indicate that the driver can access **all the
+features** of that specific version of MongoDB server unless those features 
+have been deprecated or removed. 
+
+If you are upgrading your server or driver version, you can use this table
+to identify the following:
+
+- version of the driver you need to access all the newest features
+  included in that version of the server
+
+- versions of the drivers tested against versions of the server
```

Comment:
What do we mean by "tested" here. Is it "MongoDB currently tests" or somesuch? Or is it more like "this driver version would pass all integration tests with this server version"?

Reply:
Last time I spoke with people on the drivers teams (though not all were present), they mentioned they were tested with those versions, but not necessarily using evergreen nor are they still tested. I'm guessing all the point releases get tested, but maybe not with the same versions of server. Maybe I'll leave it out for now and convert this into a sentence rather than a list of one.